index.js

```diff
@@ -43,6 +43,72 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Sanctuary started as a complementary project to [Ramda][], aiming to
+//. provide some of Ramda's functionality in a "safer" way, by utilizing
```

Comment:
How about _Sanctuary was conceived as a complementary project to [Ramda][], aiming to provide some of its functionality in a safer way_ to avoid repeating the word Ramda twice in quick succession.